src/content/ui-api/monitoring/index.md

```diff
@@ -0,0 +1,66 @@
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+# Monitoring and dashboards
+
+Giant Swarm runs a monitoring stack in each installation, based on [Prometheus](https://prometheus.io/), collecting data from most management cluster components, as well as many workload cluster metrics.
+
+For accessing these metrics, both for you as customers and for our staff we provide a [Grafana](https://grafana.com/) instance per installation.
+
+![Home dashboard](home-dashboard.png)
+
+We aim to provide a useful set of dashboards for you to quickly explore all metrics relevant to you, giving you access to the same information that we use for operations, support, and billing.
+
+## Accessing dashboards
+
+If you have access to the Giant Swarm [web interface]({{< relref "/ui-api/web" >}}), the easiest way to get to Grafana is to click the link **Monitoring** in the top menu when logged in.
+
+In case you don't have access to the Giant Swarm web interface, you can ask your Giant Swarm support contact for the URL.
+
+In case you want to construct the URL to the Grafana web application in your installation yourself, it follows the same pattern as the one for our web interface and many other services.
+
+For example, if your web interface URL is
+
+    https://happa.g8s.gollum.westeurope.azure.gigantic.io/
+
+then simply replace `happa` with `grafana` to get to your Grafana URL:
+
+    https://grafana.g8s.gollum.westeurope.azure.gigantic.io/
+
+## Authentication and authorization
+
+Access to Grafana is controlled via **single sign-on** (SSO), using the same identity provider you use for the [Management API]({{< relref "/ui-api/management-api/authentication" >}}).
+
+In order to access Grafana as a customer, you must be a member of the admin group. Specifically, that's a group in your identity provider selected to specify which users have admin permissions in the Giant Swarm installation.
+
+**Note**: If you don't have SSO configured yet or have any questions regarding the admin group, please contact your Account Engineer.
+
+Once you open the Grafana URL for your installation, you will be greeted by a page titled "Log in to dex", where you are prompted to select the identity provider to authenticate with. The screenshot below shows an example.
+
+![Selecting an OIDC provider](access.png)
+
+There is one identity provider configured for Giant Swarm staff and one for you as a customer admin. Depending on the type of identity provider used for you as a customer, the label for the button to click can look slightly different.
+
+After selecting the right identity provider, you may run through an authentication workflow. However, if you are already authenticated within the current browser, this will be skipped and you should see the Home dashboard as a result.
+
+## Repositories
+
+Grafana is deployed to Giant Swarm management clusters via our own App, to be found in the [giantswarm/g8s-grafana](https://github.com/giantswarm/g8s-grafana) Github repository.
```

Comment:
Is it useful to link to a private repo?

Reply:
No. I missed that part. Will remove that bit.